docs/index.rst

```diff
@@ -117,69 +126,6 @@ nature, you get a fast DataFrame at 1MB and 1TB+.
 you can find 'Getting started' guide, system architecture and internal implementation details docs
 and lots of other useful information.**
```

Comment:
Renamed to "Developer" Section. "Getting started" is not part of the Developer section. Would be good to link to the pages here directly.